src/mtev_console.c

```diff
@@ -79,20 +79,51 @@
 #include "noitedit/fcns.h"
 #include "noitedit/map.h"
 
+#define OL(a) do { \
+  pthread_mutex_lock(&(a)->outbuf_lock); \
+} while(0)
+#define OUL(a) do { \
+  pthread_mutex_unlock(&(a)->outbuf_lock); \
+} while(0)
+
+static int
+nc_write_with_lock(mtev_console_closure_t ncct, const void *buf, int len) {
```

Comment:
Nit: Naming of this function is confusing, to me. We are not locking. We assume the lock is being hold by the caller.

Reply:
This is fairly consistent.  The caller call the _with_lock function with a lock.